lizmap/project_checker_tools.py

```diff
@@ -379,11 +382,14 @@ def trailing_layer_group_name(layer_tree: QgsLayerTreeNode, project, results: Li
     return results
 
 
-def authcfg_url_parameters(datasource: str) -> bool:
+def french_geopf_authcfg_url_parameters(datasource: str) -> bool:
     """ Check for authcfg in a datasource, using a plain string.
 
     This function is not using QgsDataSourceUri::authConfigId()
     """
+    if 'data.geopf.fr' not in datasource:
```

Comment:
With this , any layer from other source than geoplateforme will accept autcfg param ?

Reply:
Thanks, good question.

Because `authcfg` can work with someone hosting LWC on its own server, if he sets the master password with the QGIS auth database : https://docs.qgis.org/3.28/en/docs/user_manual/auth_system/auth_workflows.html#qgis-server-support

In my mind, I wanted to prioritize first the error for French users first, by providing the link and explaining that they should follow the doc for QGIS 3.22.

The check will be done only for : 

* people having checked the "Prevent using the AUTH database"
* people on Lizmap Cloud plateform